### DISCUSSION TITLE
Select the highest weight for 2 hands

### DISCUSSION THREAD

**Discussion by andupotorac** [Bing-su/adetailer#134] [Ideas]
So for example here:
![image](https://github.com/Bing-su/adetailer/assets/3269738/4a497cd4-1078-4ce7-9744-c9c9c7b24603)

It would only work on the .77 and .69 values, and ignore the rest of the false positives.

Also wondering if it could remove false positives by itself. For example, had to remove that one and inpaint it myself after the image was ready:

![image](https://github.com/Bing-su/adetailer/assets/3269738/5c51a1fe-3a46-4dae-a0fb-0194fe2e708d)

**Comment by Bing-su** [comment]
![Cap 2023-06-20 10-42-49-731](https://github.com/Bing-su/adetailer/assets/37621276/e6cb6f45-32b3-4521-ba5f-30244e62702d)
Try adjusting the detection model confidence threshold.

**Reply by andupotorac** [reply]
Yep went with .55 and made that default in config, thanks!
